opentech/static_src/src/app/src/redux/reducers/notes.js

```diff
@@ -43,8 +45,44 @@ function note(state, action) {
     }
 }
 
+function notesCreating(state = [], action) {
+    switch (action.type) {
+        case START_CREATING_NOTE_FOR_SUBMISSION:
+            return [
+                ...state,
+                action.submissionID,
+            ];
+        case UPDATE_NOTE:
+        case FAIL_CREATING_NOTE_FOR_SUBMISSION:
+            return state.filter(v => v !== action.submissionID);
+        default:
+            return state
+    }
+}
+
+
+function notesFailedCreating(state = {}, action) {
```

Comment:
I'm not sure how i feel about these two stores. They look like they work. But are they the best pattern. Not sure...

Reply:
Should we add those to a particular submission object rather than having a separate store?

Reply:
To be honest it is probably OK. What i'm trying to work out is the case:
> If we wanted to prevent changing the current submissions because we are performing an action on it - where would we be looking for the data? 

Perhaps it is ok where it is.